src/network/Main.java

```diff
@@ -106,4 +112,18 @@ public static void main(String args[]) {
 		}
 		in.close();
 	}
+	
+	/*
+	 * returns the average distance from centroid to members of the cluster
+	 */
+	private static double calcAverageDistance(Cluster cluster) {
+		double averageDistance = 0;
+		for(int i = 0; i < cluster.getMembers().size(); i++) {
+			for(int j = 0; j < cluster.getMembers().size(); j++) {
+				averageDistance += cluster.getMembers().get(i).calcDistance(cluster.getMembers().get(j));
```

Comment:
This looks like it is doing the average distance between members of the cluster rather than distance from the center.

Reply:
Were we evaluating it with average distance from the center or average distance between members of the cluster?

Reply:
I think distance from the center makes more sense but we could probably do it either way